test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -1017,5 +1017,19 @@ private static bool IsProjectJson(string configFileName)
             // Simply test the extension as that is all we care about
             return string.Equals(Path.GetExtension(configFileName), ".json", StringComparison.OrdinalIgnoreCase);
         }
+
+        public static string CreateDummyConfigFile(string directoryPath)
```

Comment:
Does this need to be in a shared location? Can't it just be in the unit test class above?